data/json/mutations.json

```diff
@@ -2521,15 +2538,151 @@
         "type" : "mutation",
         "id" : "GIZZARD",
         "name" : "Gizzard",
-        "points" : -5,
-        "description" : "You seem to get full faster now, and food goes through you more rapidly as well.",
+        "points" : 2,
+        "description" : "Your stomach has split into a smaller stomach, and a gizzard. Slightly increases the energy gained through eating, and prevents you from eating as much.",
+        "valid" : false,
+        "purifiable" : false,
+        "prereqs" : ["BEAK", "BEAK_PECK", "BEAK_HUM", "BEAK_FLAT"],
+        "leads_to" : ["BIRDCROP"],
+        "threshreq" : ["THRESH_BIRD"],
+        "category" : ["BIRD"]
+    },{
+        "type" : "mutation",
+        "id" : "WINGS_FOLDING",
+        "name" : "Folding Wings",
+        "points" : 2,
         "valid" : false,
+        "visibility" : 2,
+        "ugliness" : 2,
+        "description" : "Your wings have become more flexible, allowing you to fold them against your body. They no longer encumber you, and negate fall damage.",
+        "types" : ["WINGS"],
         "purifiable" : false,
-        "prereqs" : ["BEAK", "BEAK_PECK", "BEAK_HUM"],
-        "prereqs2" : ["LIGHTEATER"],
+        "prereqs" : ["WINGS_BIRD"],
         "threshreq" : ["THRESH_BIRD"],
-        "cancels" : ["GOURMAND"],
+        "armor" : [ { "parts" : "TORSO", "bash" : 2 } ],
         "category" : ["BIRD"]
+    },{
+        "type" : "mutation",
+        "id" : "BIRDCROP",
+        "name" : "Bird Crop",
+        "points" : 3,
+        "description" : "Your body has grown a bird crop, allowing you to store excess food if you overeat.",
+        "valid" : false,
+        "purifiable" : false,
+        "prereqs" : ["GIZZARD"],
+        "threshreq" : ["THRESH_BIRD"],
+        "category" : ["BIRD"]
+    },{
+        "type" : "mutation",
+        "id" : "BIRDSONG1",
+        "name" : "Lilting Voice",
+        "points" : 1,
+        "valid" : false,
+        "description" : "You have a pleasant lilt to your voice.  Persuading NPCs will be easier, but threatening them will be slightly harder.",
+       "category" : ["BIRD"],
+       "changes_to" : ["BIRDSONG2"],
+        "social_modifiers" : {
+            "persuade" : 20,
+            "intimidate" : -5
+        }
+    },{
+        "type" : "mutation",
+        "id" : "BIRDSONG2",
+        "name" : "Birdsong",
+        "points" : 2,
+        "valid" : false,
+        "description" : "Your voice is a mixture of song and words. Persuading NPCs will be much easier, but threatening them will be harder.",
```

Comment:
Double space. I also have the mental image of someone trying to intimidate someone in a high-pitched singing voice and them just shaking their head and laughing.